tests/functional/adapter/incremental/test_incremental_on_schema_change.py

```diff
@@ -251,3 +251,59 @@ def test_run_incremental_sync_all_columns_time_ingestion_partitioning(self, proj
         compare_source = "incremental_sync_all_columns_time_ingestion_partitioning"
         compare_target = "incremental_sync_all_columns_time_ingestion_partitioning_target"
         self.run_twice_and_assert(select, compare_source, compare_target, project)
+
+
+_MODELS__SRC_ARTISTS = """
+{{
+    config(
+        materialized='table',
+    )
+}}
+{% if var("version", 0) == 0 %}
+    select {{ dbt.current_timestamp() }} as inserted_at, 'john' as name
+{% else %}
+    -- add a non-zero version to the end of the command to get a different version:
+    -- --vars "{'version': 1}"
+    select {{ dbt.current_timestamp() }} as inserted_at, 'engineer' as Job,
```

Comment:
Is `Name` purposely skipped here? I noticed you had it in the `dbt-tests-adapter` version.

Reply:
yeah for bigquery when ran it seemed to count the column as already existing and would fail. removing and running seems fine even if I changed the casing of name to Name it still failed. for column already existing happy to look into it further.

my thought was that since bigquery checks column casing natively it sees them as the same thing so nothing new to add and that seems to borq the query